app/lib/onboarding/v2/onboarding_categories_development.yml

```diff
@@ -27,3 +275,6 @@
     - account: '@jtomchak@moth.social'
       type: account
       summary: Engineer at Mammoth.
+    - account: '@pxl@moth.social'
+      type: account
+      summary: Designer at Mammoth.
```

Comment:
I don't want to add pxl unless we have confirmation from him. (I know Bart asked, and I think he said "not yet", or… something).

Reply:
only for dev 👍